btk/draw_blends.py

```diff
@@ -278,38 +293,45 @@ def run_mini_batch(self, blend_list, obs_conds):
             blend_list: List of catalogs with entries corresponding to one blend.
             obs_conds (list): List of `descwl.survey.Survey` class describing
                 observing conditions in different bands.
+            survey_name (str): Name of the survey (see obs_conditions.py for
+                                currently available surveys)
 
         Returns:
             `numpy.ndarray` of blend images and isolated galaxy images, along with
             list of blend catalogs.
         """
         mini_batch_outputs = []
         for i in range(len(blend_list)):
-            dx, dy = get_center_in_pixels(
-                blend_list[i], self.stamp_size, self.pixel_scale
-            )
+            pixel_scale = obs_conds[0].pixel_scale
+            dx, dy = get_center_in_pixels(blend_list[i], obs_conds[0].wcs)
```

Comment:
Pixel_scale/WCS are always the same for a given survey right? We might want to restructure the cutout class or something to make this explicit.

Reply:
Hmm yeah, I agree that probably better off just having different surveys for these cases. Let me take a stab at the code to see if I can simplify the logic a little.